casper/src/main/scala/coop/rchain/casper/MultiParentCasperImpl.scala

```diff
@@ -185,28 +186,20 @@ class MultiParentCasperImpl[F[_]: Sync: Concurrent: Log: Time: SafetyOracle: Las
     * @param deployData
     * @return
     */
-  private def validateDeploy(deployData: DeployData): Either[DeployError, Unit] = deployData match {
-    case d if (d.sig == ByteString.EMPTY)      => missingSignature.asLeft
-    case d if (d.sigAlgorithm == "")           => missingSignatureAlgorithm.asLeft
-    case d if (d.deployer == ByteString.EMPTY) => missingUser.asLeft
-    case _ =>
-      val maybeVerified = SignDeployment.verify(deployData)
-      maybeVerified.fold[Either[DeployError, Unit]](
-        unknownSignatureAlgorithm(deployData.sigAlgorithm).asLeft
-      ) {
-        case false => signatureVerificationFailed.asLeft
-        case true  => ().asRight
-      }
-  }
+  private def validateDeploy(deployData: DeployData): Either[DeployError, Unit] =
```

Comment:
`Either.cond` is a bit less verbose

Reply:
This code is removed in later commits, when deployer is removed from DeployData.